README.md

```diff
@@ -17,34 +17,27 @@ If you want to test the current version (which requires php 7.3) :
 ```bash
 $ composer install
 $ docker-compose up -d
-$ symfony php bin/console doctrine:migrations:load
-$ symfony php bin/console doctrine:fixtures:load
+$ php bin/console doctrine:migrations:load
```

Comment:
Actually there is no migration file, and the command is `doctrine:migrations:migrate`, also the `symfony` binary is fine because you need it for `serve` anyway, but it would be nice to mention that it need to be installed.

From a new install, `symfony php bin/console doctrine:schema:create` worked fine for me.

Reply:
👍 

de plus `symfony php` est nécessaire pour utiliser la version de php utilisée par le projet et non forcément celle par défaut.